apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -463,7 +463,7 @@ fields("node") ->
             )},
         {"config_files",
             sc(
-                list(string()),
```

Comment:
when we config `node.config_files = ["test"]`
```
node {
  name = "emqx@127.0.0.1"
  cookie = "emqxsecretcookie"
  data_dir = "data"
  config_files = ["test"]
}
```
the config schema check will failed by:
```
exception throw: {emqx_conf_schema,[#{kind => validation_error,
                                      path => "node.config_files",
                                      reason =>
                                          #{expected_type => <<"[string()]">>},
                                      value => [<<"test">>]}]}
  in function  hocon_tconf:assert_no_error/2 (hocon_tconf.erl, line 1142)
  in call from hocon_tconf:map/4 (hocon_tconf.erl, line 295)
  in call from hocon_cli:get_values/3 (hocon_cli.erl, line 194)
  in call from hocon_cli:multi_get/2 (hocon_cli.erl, line 182)
  in call from escript:run/2 (escript.erl, line 750)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1
  in call from init:do_boot/3
```
So we change the type to `hocon:array(string()).`